db/data_migration/20150511162040_reclassify_correspondence.rb

```diff
@@ -0,0 +1,24 @@
+PUBLISHED_AND_PUBLISHABLE_STATES = %w(published draft archived submitted rejected scheduled)
+
+correspondence_scope = Publication.
+  where(state: PUBLISHED_AND_PUBLISHABLE_STATES).
+  where(publication_type_id: PublicationType::Correspondence.id)
+
+count = correspondence_scope.count
+index = 0
+reclassified_count = 0
+
+puts "Checking the political status of #{count} correspondence"
```

Comment:
"correspondence" is only plural as an uncountable noun, you want "correspondences" here, not that it matters for the sake of this PR.